images/fcos-bastion-image/root/usr/lib/systemd/system/nginx.service

```diff
@@ -0,0 +1,23 @@
+[Unit]
+Description=Creates a nginx container service
+Wants=network-online.target
+After=network-online.target
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/usr/share/bastion_services/nginx/nginx_systemd.conf
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run -d --name nginx-container -p 80:80/tcp -p 443:443/tcp \
+    -v ${PODMAN_NGINX_BASEDIR}/html:/usr/share/nginx/html \
```

Comment:
@aleskandro here I have already added html folder, what should be the desire folder structure `/var/opt/html/html` or `/var/opt/html/`?

Reply:
I have made the changes as `/var/opt/html/`.